### PR TITLE
Fix validate-module sanity tests

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/become/enable.py [hunk 1]
```diff
@@ -6,8 +6,9 @@
 __metaclass__ = type
 
 DOCUMENTATION = """
-author: Ansible Networking Team
-become: enable
+author:
+ - Ansible Networking Team (@network-team)
```
Comment (NilashishC) [inline plugins/become/enable.py:10]: @network-team doesn't seem to be an actual GitHub id.

**Review by NilashishC — CHANGES_REQUESTED** [review]

File: plugins/become/enable.py (hunk 1 above)
Comment (ashwini-mhatre) [reply plugins/become/enable.py]: ack

File: docs/ansible.netcommon.libssh_connection.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ ansible.netcommon.libssh
 **(Tech preview) Run tasks using libssh for ssh connection**
 
 
-Version added: 2.10
+Version added: 2.6.0
```
Comment (NilashishC) [inline docs/ansible.netcommon.libssh_connection.rst:11]: Shouldn't this be `1.0.0`?

File: plugins/connection/libssh.py [hunk 3]
```diff
@@ -5,14 +5,15 @@
...
+    author:
+      - Ansible Networking Team (@ansible-network)
+    name: libssh
     short_description: (Tech preview) Run tasks using libssh for ssh connection
     description:
         - Use the ansible-pylibssh python bindings to connect to targets
         - The python bindings use libssh C library (https://www.libssh.org/) to connect to targets
         - This plugin borrows a lot of settings from the ssh plugin as they both cover the same protocol.
-    version_added: "2.10"
+    version_added: 2.6.0
```
Comment (NilashishC) [inline plugins/connection/libssh.py:16]: Same as above.

File: docs/ansible.netcommon.libssh_connection.rst (hunk 2 above)
Comment (NilashishC) [reply docs/ansible.netcommon.libssh_connection.rst:11]: https://github.com/ansible-collections/ansible.netcommon/blob/main/CHANGELOG.rst#v110

**Comment by GomathiselviS** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Pull request merged**